### DISCUSSION TITLE
how to make lspatch work with modules that configuration app?

### DISCUSSION THREAD

**Discussion by soredake** [LSPosed/LSPatch#128] [Q&A]
I've tried lspatch with "android faker" module that have configuration app. Is it possible to make lspatch work with type of modules?

![Без имени](https://user-images.githubusercontent.com/5204968/198877171-90a83d5f-1ed5-4764-b196-ffc92f37037e.jpg)

**Comment by slopsommelier** [comment]
Same problem. GPS Setter can't send fake GPS to my app because the configuration app doesn't work.
